StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/DeferredIntentContext.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  DeferredIntentContext.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 3/3/23.
+//
+
+import Foundation
+import StripePayments
+
+class DeferredIntentContext {
```

Comment:
In terms of code organization, wdyt about making a `PaymentSheet+DeferredAPI.swift` file or something and putting this and all the deferred helper methods in there?